### DISCUSSION TITLE
which one has lower training cost?

### DISCUSSION THREAD

**Discussion by zmy-08** [PixArt-alpha/PixArt-alpha#143] [Q&A]
Hello, compared with pixArt-LCM and pixArt-α, which one is suitable for training your own data set and has lower training cost? Because pixArt-α can be trained on my 3090, but PixArt-LCM cannot be trained.

**Comment by lawrence-cj** [comment]
Oh, PixArt-LCM is finetuned based on PixArt-α. You can just further finetune on the α model. It will be fine.
